### DISCUSSION TITLE
Logo looks extremely similar to AWS Documentation logo

### DISCUSSION THREAD

**Discussion by ghost** [opentofu/opentofu#858] [General]
Hi,

Has anyone noticed OpenTofu's logo looks very similar to the logo for the AWS Documentation webpage? 

Just check them out side-by-side in browser tabs:

![image](https://github.com/opentofu/opentofu/assets/65383009/df7abe80-d6bb-4381-b4bd-d5a7445257fa)

Considering the fact that many OpenTofu users are likely to have tabs open from both these sites simultaneously, this seems like a tiny oversight.

Oh, my colleague also made this: 
![image](https://github.com/opentofu/opentofu/assets/65383009/88337650-20dd-458a-abc3-593a05f299fb)

**Comment by rdhar** [comment]
Has the li'l Tofu eyes always been facing to the left? 

I think I Mandela Effect'ed myself into thinking that it used to face right... or maybe it's the dancing gif of it? 

![dancing-tofu](https://github.com/opentofu/opentofu/assets/19497993/40090c6a-349c-44fc-bc05-636853eb5674)
